components/Footer/Footer.tsx

```diff
@@ -0,0 +1,44 @@
+import Image from 'next/image';
+
+import { footerData } from '@/constant';
+
+export default function Footer() {
+	return (
+		<footer>
+			<div className=' w-full border-t bg-white  dark:border-t-black-300 dark:bg-black-200'>
+				<div className='flex  h-full flex-col items-center gap-4 px-3 py-4 sm:h-20  sm:flex-row sm:justify-between sm:px-8'>
+					<div className='text-center text-xs text-white-500 dark:text-white sm:text-sm md:text-lg'>
+						<p className='whitespace-nowrap'>
+							© 2023 Wirayuda. <span>All rights reserved.</span>
+						</p>
+					</div>
+					<div className='inline-flex gap-6'>
+						{footerData.map((item) => (
+							<a
```

Comment:
In Next.js, it's better to use the Link component from Next for internal links instead of anchor tags.

`<Link> `is a React component that extends the HTML `<a>` element to provide [prefetching] and client-side navigation between routes. It is the primary way to navigate between routes in Next.js. You can read more about it [here](https://nextjs.org/docs/app/api-reference/components/link).